### DISCUSSION TITLE
Is there a way to exclude projects already supported by large corporations?

### DISCUSSION THREAD

**Discussion by ieatfood** [stackaid/stackaid#1] [Q&A] [answered]
I would feel like my donation would make more impact if a large portion of it didn't go to repos in Microsoft or Facebook orgs.

**Comment by voxpelli** [comment, marked as answer]
There is now it appears like: https://www.stackaid.us/blog/managing-funded-organizations

I just opted out Microsoft, AWS and some others from my account
